Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -305,6 +308,45 @@ namespace AzFramework
         }
         return SpawnableInstanceDescriptor::GetInvalidDescriptor();
     }
+
+    void SpawnableEntitiesManager::DespawnAllEntitiesInTicketByEntityID(const AZ::EntityId& entityId, DespawnAllEntitiesOptionalArgs optionalArgs)
+    {
+        if (entityId.IsValid())
+        {
+            EntitySpawnTicket::Id ticketId = 0;
```

Comment:
Maybe something like `const EntitySpawnTicket::Id InvalidTicketId = -1;` and use it as a named constant here and below